### PR TITLE
Add Tutorial for TamTam Micro Activity

### PR DESCRIPTION
**Pull request by AndreaGon**
TamTam Micro Activity should now have a tutorial:

tamtam

### CODE REVIEW

**Comment by quozl** [comment]
Well done, I like it.

File: activities/TamTamMicro.activity/css/activity.css [hunk 1]
```diff
@@ -20,6 +24,135 @@
   z-index: 5;
 }
 
+/*Tutorial CSS*/
+.tutorial-title {
+	background-color: #808080 !important;
```
Comment (llaske) [inline activities/TamTamMicro.activity/css/activity.css]: Could you avoid mixing different type of indentation? Here indentation should be white spaces.

**Comment by AndreaGon** [comment]
This should fix the indentation issues

**Comment by llaske** [comment]
There is an issue on localization, whatever the language settings in Sugarizer, it's always the language which is used. Look like webl10n library is not initialized in the activity like it should be https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step5.md#initialize-localization

**Comment by AndreaGon** [comment]
Hmmm, that's weird, as base on the gif above, I've set the Sugarizer language to French and it worked.

I will look into this.

**Comment by AndreaGon** [comment]
@llaske It turns out that I did not add the webL10n initialization from my local work environment into my commits. This new commit should hopefully fix the issue.

Let me know what you think!

**Comment by llaske** [comment]
Nice. It works now. Thanks.

**Pull request merged**